src/components/RangeSlider/RangeSlider.tsx

```diff
@@ -34,16 +34,12 @@ export const RangeSlider = forwardRef<HTMLInputElement, RangeSliderProps>(
 
     return (
       <>
-        <div data-testid="flowbite-range-slider" className={twMerge(theme.root.base, className)}>
-          <div className={theme.field.base}>
-            <input
-              ref={ref}
-              type="range"
-              className={twMerge(theme.field.input.base, theme.field.input.sizes[sizing])}
-              {...props}
-            />
-          </div>
-        </div>
+        <input
+          ref={ref}
```

Comment:
Do we need [forwardRef](https://react.dev/reference/react/forwardRef) for this also?

Reply:
@tulup-conner I didn't quite understand your question. I believe, if we want `<RangeSlider />` to support refs, we do, don't we? Or is this about whether we need this functionality in the first place?

Reply:
No that's my fault, I only looked at the diff and didn't notice this already has forwardRef